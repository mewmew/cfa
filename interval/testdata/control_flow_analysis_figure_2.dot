// Derived from figure 2 in F. Allen, "Control Flow Analysis", 1970.
//
// http://www.cs.columbia.edu/~suman/secure_sw_devel/p1-allen.pdf

digraph G {
	// Nodes.
	1 [label=entry];
	2;
	3;
	4;
	5;
	6;
	7;
	8;

	// Edges.
	1 -> 2
	2 -> 3
	2 -> 7
	3 -> 4
	3 -> 5
	4 -> 3
	4 -> 6
	5 -> 6
	6 -> 7
	7 -> 2
	7 -> 8
}
