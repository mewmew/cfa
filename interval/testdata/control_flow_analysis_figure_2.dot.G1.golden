digraph G1 {
	// Node definitions.
	1 [label=entry];
	2;
	3;
	4;
	5;
	6;
	7;
	8;

	// Edge definitions.
	1 -> 2;
	2 -> 3;
	2 -> 7;
	3 -> 4;
	3 -> 5;
	4 -> 3;
	4 -> 6;
	5 -> 6;
	6 -> 7;
	7 -> 2;
	7 -> 8;
}
